### DISCUSSION TITLE
Use AirConnect to support AirPlay 2?

### DISCUSSION THREAD

**Discussion by thawks** [philippe44/AirConnect#340] [Q&A] [answered]
Hi! First, this is an incredible project, just used it to convert an old Sonos I have to Airplay, years after unwittingly buying a play:1 (rather than a one) from a friend. Deeply grateful. 

Long term, I'd like to hack together a multi-room set up with the Sonos', a Mac mini and hifi system. Does AirConnect support Airplay 2 for the Sonos speakers on the network? and can it be used to turn the Mac mini into an airplay receiver? 

Thanks!!

**Comment by pwt** [comment, marked as answer]
You might want to take a look at the following, which discuss AirPlay 2 support: #258, #218, #114, #97, #88, #79, #53.

The bottom line is that it's (probably) technically not possible to support AirPlay 2 with a 'bridge' approach like that offered by AirConnect, because it's not possible to control the end-to-end synchronisation required by the protocol.

You may find other software that can act as an AirPlay 2 target for use with a Mac with directly attached speakers. That *is* technically possible, because there's no requirement for downstream synchronisation with remote speakers.

**Comment by thawks** [comment]
Got it, thanks for the info. Sorry for not searching the issues page! -T

> On Mar 29, 2021, at 10:50 AM, pwt ***@***.***> wrote:
> 
> 
> You might want to take a look at the following, which discuss AirPlay 2 support: #258 <https://github.com/philippe44/AirConnect/issues/258>, #218 <https://github.com/philippe44/AirConnect/issues/218>, #114 <https://github.com/philippe44/AirConnect/issues/114>, #97 <https://github.com/philippe44/AirConnect/issues/97>, #88 <https://github.com/philippe44/AirConnect/issues/88>, #79 <https://github.com/philippe44/AirConnect/issues/79>, #53 <https://github.com/philippe44/AirConnect/issues/53>.
> 
> The bottom line is that it's (probably) technically not possible to support AirPlay 2 with a 'bridge' approach like that offered by AirConnect, because it's not possible to control the end-to-end synchronisation required by the protocol.
> 
> You may find other software that can act as an AirPlay 2 target for use with a Mac with directly attached speakers. That is technically possible, because there's no requirement for downstream synchronisation with remote speakers.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/philippe44/AirConnect/discussions/340#discussioncomment-544242>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AAD7STFXJFOE5U4F3MUEJRDTGCOTBANCNFSM4Z7XISDQ>.
> 

—
Tyler Hawks
(414) 530-4295
